packages/ui/src/components/pages/SetupWallet/NewWallet/BackupWallet.tsx

```diff
@@ -0,0 +1,131 @@
+import { ChangeEvent, FormEvent, useMemo, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { useDispatch, useSelector } from 'react-redux';
+import { useEffectOnce } from 'react-use';
+import { generateMnemonic } from '@polkadot/util-crypto/mnemonic/bip39';
+import { WalletBackup } from '@coong/keyring/types';
+import { LoadingButton } from '@mui/lab';
+import { Button, Checkbox, FormControlLabel, FormGroup } from '@mui/material';
+import BackupSecretRecoveryPhrase from 'components/pages/SetupWallet/NewWallet/BackupSecretRecoveryPhrase';
+import JsonFile from 'components/shared/export/JsonFile';
+import QrCode from 'components/shared/export/QrCode';
+import CryptoJS from 'crypto-js';
+import useSetupWallet from 'hooks/wallet/useSetupWallet';
+import { setupWalletActions } from 'redux/slices/setup-wallet';
+import { RootState } from 'redux/store';
+import { Props, TransferableObject, WalletRecoveryMethod } from 'types';
+
+interface BackupInstructionProps extends Props {
+  method: WalletRecoveryMethod;
+}
+
+function BackupInstruction({ method }: BackupInstructionProps): JSX.Element {
+  const { t } = useTranslation();
+  const { password, secretPhrase } = useSelector((state: RootState) => state.setupWallet);
+
+  // Preventing re-create the wallet backup when the component is re-rendered
+  const walletBackup = useMemo(() => {
+    if (!secretPhrase || !password) return;
```

Comment:
I'm in favor of us throwing an exception here instead of not render anything if there is no `secretPhrase` or `password`.